docs/src/apireference.md

```diff
@@ -182,11 +182,26 @@ delete(::ModelLike, ::Index)
 
 ### Variables
 
-Functions for adding variables. For deleting, see index types section.
+*Free variables* are the variables created with [`add_variable`](@ref) or
+[`add_variables`](@ref) and never constrained while *constrained variables* are
+the variables either created with [`add_constrained_variable`](@ref) or
+[`add_constrained_variables`](@ref) or created with [`add_variable`](@ref) or
+[`add_variables`](@ref) and constrained after being created using
```

Comment:
For a constrained variable, is it valid to ask what set is constraining the variable? This is nonsymmetric because you can add multiple `SingleVariable` constraints on the same variable using `add_constraint`, but only one when using `add_constrained_variable`.

Reply:
No you cannot it may be a relevant next step. That would allow `copy_to` to copy variables with the same set they were added. Currently on bl/variable_bridges, if a variable is constrained twice then it's not determined by the order of creation but rather by the type of set

Reply:
What if the definition of "constrained variable" is changed to one that is created with `add_constrained_variable`? This would make it simpler in the future to implement this change.
Then the current `copy_to` behavior would be a bit quirky but not a big deal.

Reply:
I don't consider this comment resolved.

Reply:
Oops, didn't see your comment. Good idea, I have updated the PR